domain/domain.go

```diff
@@ -35,14 +35,15 @@ var ddlLastReloadSchemaTS = "ddl_last_reload_schema_ts"
 // Domain represents a storage space. Different domains can use the same database name.
 // Multiple domains can be used in parallel without synchronization.
 type Domain struct {
-	store        kv.Storage
-	infoHandle   *infoschema.Handle
-	ddl          ddl.DDL
-	leaseCh      chan time.Duration
-	lastLeaseTS  int64 // nano seconds
-	m            sync.Mutex
-	txnTS        uint64 // It's used for checking schema validity.
-	IsMockFailed bool   // It mocks reload failed.
+	store            kv.Storage
+	infoHandle       *infoschema.Handle
+	ddl              ddl.DDL
+	leaseCh          chan time.Duration
+	lastLeaseTS      int64 // nano seconds
+	m                sync.Mutex
+	schemaValidity   *schemaValidityInfo
+	txnTS            uint64 // It's used for checking schema validity.
+	MockReloadFailed bool   // It mocks reload failed.
```

Comment:
`schemaValidityInfo` field can be just struct rather than pointer, and we can put `txnTS` and `MockReloadFailed` in `schemaValidityInfo`.